src/LinAlg/SAM.C

```diff
@@ -809,7 +809,8 @@ bool SAM::applyDirichlet (Vector& dofVec) const
     {
       int ip = mpmceq[iceq-1];
       dofVec[idof] = ttcc[ip-1];
-    }
+    } else if (iceq == 0)
+      dofVec[idof] = 0.0;
```

Comment:
Have to think about this one, when I have the source code in front of me. It looks OK, but I can't see what this has to do with mixed problems? Looks more like a general error (if it is an error). iceq is zero when a DOF is fixed and I though in this case it should be necessary to do anything since dofVec should already have a zero value. Or can you explain why this is not so, only when doing mixed.